src/tn/eduVision/entités/Admin.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package tn.eduVision.entités;
+
+import java.math.BigDecimal;
+
+/**
+ *
+ * @author Sayf
+ */
+public class Admin extends Utilisateur{
```

Comment:
The classes should all impliment IEntity i think ama not a big issue we can get rif of it cause we won't use DI

Reply:
i dont think IEntity is needed right now because we dont have certain rules for the entity to define for the interface